### ISSUE TITLE
How long to train a model without overfitting?

### ISSUE THREAD

**Issue by Crista23** [OpenNMT/OpenNMT-py#1145]
I am unsure what loss to look at to decide how long to leave the model training. When using Tensorboard to determine the training and validation accuracy, I can see them in different plots (see [training accuracy](https://ibb.co/gZ5JGPk) and [validation accuracy](https://ibb.co/PtJjRKg) for the machine translation data and tutorial. Is it possible to plot these two in the same graph? And is there any early stopping criterion implemented to stop the model from overfitting? At the same time I am confused with the number of epoch and number of training steps - as it seems now in the plot there is on the x-axis the number of training steps and not the epochs. And finally, how often is the model validated and why the x-axis for the training and validation plots does not have the same number of steps? Thank you!

**Comment by vince62s** [comment]
Hi,
Please copy this question to the forum, where you will find someone answering but also you will find many already answered similar questions there. (there is a tuto on Tensorboard)
Just answering one: there is no stopping criterion. You need to check the Validation PPL/ACC when it has converged.
Cheers.
